### PR TITLE
Development documentation

### PR DESCRIPTION
**Pull request by dunossauro**

### CODE REVIEW

File: docs/develop.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# development
+
+If you like Tigre purpose and would like to contribute this page can help you.
+
+Tigre support only python3.7+ and you need version 3.7+ to develop. We recommend that you use `Pyenv` to manage many python enviroments. Basic instructions can be found below.
```
Comment (ThiagoDiasV) [inline docs/develop.md]: Some changings to review:

`Tigre support` -> `Tigre supports`

`python3.7+` -> `Python 3.7+`

`python environments.` -> `Python environments.`

File: docs/develop.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+For develop we recommend `Poetry` and configuration can be found below too.
```
Comment (ThiagoDiasV) [inline docs/develop.md]: Here I guess `For development` is more correct than `For develop`

File: docs/develop.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+To manipulate remote webdrivers, we has a `docker_test_image` on own reporsitory, but we recommends `Selenoid` containers. Are more stable
```
Comment (ThiagoDiasV) [inline docs/develop.md]: `on own reporsitory` -> `on this repository`

`we recommends` -> `we recommend`

`Are more stable` -> `They are more stable.`

File: docs/develop.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+## Pyenv
+
+#### Installing Pyenv
+
+To install Pyenv, follow the [Pyenv repository](https://github.com/pyenv/pyenv) instructions to Unix-like, or [pyenv-win](https://github.com/pyenv-win/pyenv-win) to windows enviroments.
```
Comment (ThiagoDiasV) [inline docs/develop.md]: `windows enviroments.` -> `Windows environments.`

File: docs/develop.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
+# development
```
Comment (ThiagoDiasV) [inline docs/develop.md]: `Development` instead of `development`? With the first letter capital.

File: docs/index.md [hunk 6]
```diff
@@ -1,13 +1,12 @@
 # Tigre
 
-Tiger was born to ease the confusing calls of selenium webdriver.
-
-If you have problems with different APIs for calling remote, local or mobile browsers. If you are confused by so many different capabilities settings or have trouble with hard-to-maintain code. Tiger was designed especially for you.
+Tigre was born to ease the confusing calls of selenium webdriver.
 
+If you have problems with different APIs for calling remote, local or mobile browsers. If you are confused by so many different capabilities settings or have trouble with hard-to-maintain code. Tigre was designed especially for you.
```
Comment (ThiagoDiasV) [inline docs/index.md]: If you have problems with different APIs for calling remote, local or mobile browsers, or if you are confused by so many different capabilities settings or have trouble with hard-to-maintain code, Tigre was designed especially for you.

**Review by ThiagoDiasV — CHANGES_REQUESTED** [review]
Review with a few suggestions.

File: docs/develop.md (hunk 3 above)
Comment (rvmoura96) [reply docs/develop.md]: We has a docker_test_image => we have

**Comment by dunossauro** [comment]
@rvmoura96 and @ThiagoDiasV, PR up-to-date with sugestions.

**Pull request merged**